aspnetcore/blazor/components/index.md

```diff
@@ -268,17 +268,141 @@ In the following example from the sample app, the `ParentComponent` sets the val
+  
+* To assign the result of an [explicit C# expression](xref:mvc/views/razor#explicit-razor-expressions) in the parent component to a child component's parameter, surround the expression in parentheses with an `@` symbol prefix. To calculate a date that's one week in the past for assignment to a child's parameter, use the syntax `Title="@((DateTime.Now - TimeSpan.FromDays(7)).ToString())"`, as seen in the following parent component:
+
+  ```razor
+  <ChildComponent Title="@((DateTime.Now - TimeSpan.FromDays(7)).ToString())">
+      Title from explicit Razor expression.
+  </ChildComponent>
+  ```
+
+  Use of an explicit expression to concatenate text with an expression result for assignment to a parameter is **not** supported. The following example seeks to concatenate the text "SKU-" with a product stock number (`SKU` property, "Stock Keeping Unit") provided by a parent component's `product` object. Although this syntax is supported in a Razor page (`.cshtml`), it isn't valid for assignment to the child's `Title` parameter.
+  
+  ```razor
+  <ChildComponent Title="SKU-@(product.SKU)">
+      Title from composed Razor expression. This doesn't compile.
+  </ChildComponent>
+  ```
+  
+  The code in the preceding example generates an IntelliSense error in Visual Studio and a *compiler error* if the app is built:
+  
+  > Component attributes do not support complex content (mixed C# and markup).
+  
+  To support the assignment of a composed value, use a method, field, or property. The following example performs the concatination of "SKU-" and a product's stock number in the C# method `GetTitle`:
+  
+  ```razor
+  <ChildComponent Title="@GetTitle()">
+      Composed title from method.
+  </ChildComponent>
+  
+  @code {
+      private Product product = new Product();
+
+      private string GetTitle()
+      {
+          return $"SKU-{product.SKU}";
+      }
+      
+      private class Product
+      {
+          public string SKU { get; set; } = "12345";
+      }
+  }
+  ```
   
 For more information, see [Razor syntax reference for ASP.NET Core](xref:mvc/views/razor).
 
 > [!WARNING]
 > Don't create components that write to their own *component parameters*, use a private field instead. For more information, see the [Overwritten parameters](#overwritten-parameters) section.
 
+#### Component parameters should be auto-properties
+
+Component parameters should be declared as *auto-properties*, meaning that they shouldn't contain custom logic in their getters or setters. For example, the following `StartData` property is an auto-property:
+
+```csharp
+[Parameter]
+public DateTime StartData { get; set; }
+```
+
+Don't place custom logic in the `get` or `set` accessor because component parameters are purely intended for use as a channel for a parent component to flow information to a child component. If a setter of a child component property contains logic that causes rerendering of the parent component, an *infinite rendering loop results*.
```

Comment:
I don't know why *infinite rendering loop results* is in italics here, but will leave that to you :)

Reply:
Just to emphasize that it will :boom:. I'll remove it.

```suggestion
Don't place custom logic in the `get` or `set` accessor because component parameters are purely intended for use as a channel for a parent component to flow information to a child component. If a setter of a child component property contains logic that causes rerendering of the parent component, an infinite rendering loop results.
```